packages/esm-patient-queues-app/src/visit-form/visit-form.component.tsx

```diff
@@ -112,6 +113,9 @@ const StartVisitForm: React.FC<VisitFormProps> = ({ patientUuid, toggleSearchTyp
       : [],
   );
 
+  // Check if selectedNextQueueLocation has a value selected
+  const isFormValid = selectedNextQueueLocation;
```

Comment:
i think u shd be checking if its null or undefined

Reply:
coz at a time T it might be undefined

Reply:
The goal is to just disable the button to prevent users from starting a patient visit without selecting the next location which some users are doing mistakenly. So if it's set to null or undefined it's as good as leaving it the way it's.